### DISCUSSION TITLE
Future Plans for Hoppscotch and Licensing concerns

### DISCUSSION THREAD

**Discussion by chriscz** [hoppscotch/hoppscotch#2351] [General]
Hey team Hoppscotch, thanks for building this excellent app. 

First off, a disclaimer: _I am not a lawyer_

I see your licence is MIT, which might not be ideal since it allows others to take the fruits of your labour and build a closed-source business around it. Some better alternative licences are

- A GPL licence, possibly [AGPL](https://stackoverflow.com/a/12240110/2490686) ([like Mastodon](https://github.com/mastodon/mastodon/blob/main/LICENSE)) because this is in part server-side? ([The difference between GPL 2 and 3](https://stackoverflow.com/questions/41460/what-are-the-differences-between-gpl-v2-and-gpl-v3-licenses) )
- [The Anti-Capitalist Software Licence](https://anticapitalist.software/)

To my understanding, as copyright holder you are permitted you to re-license the work, so you could always make a commercial licence for others later down the line. 

I'd love to hear the community and your thoughts on this.

**Comment by AndrewBastin** [comment]
Hi @chriscz!

Thank you for your kind words, concern and your hearty support!

Yes, the license currently is MIT and we are comfortable with it right now for the current state in which Hoppscotch is. We are planning some major revamps to the project later as we prepare to open source the entire Hoppscotch stack, on that occassion we are planning to move to a more suited license model which protects our interests.

**Reply by chriscz** [reply]
Hey @AndrewBastin 

Thanks for the prompt response, this answers my question, you may close the thread.

The best of luck to you and the team on this and other endeavours.
